configure.ac

```diff
@@ -54,6 +54,31 @@ AC_ARG_WITH([modprobedir],
                             [Path to the system modprobe.d directory])],
             [modprobedir="$withval"], [modprobedir="/lib/modprobe.d"])
 AC_SUBST(modprobedir)
+AC_ARG_WITH([dbusdir],
+            [AS_HELP_STRING([--with-dbusdir],
+                            [Path to the system dbus-1 directory])],
+            [dbusdir="$withval"], [dbusdir="/usr/share/dbus-1"])
+AC_SUBST(dbusdir)
+AC_ARG_WITH([dbussystemconfigdir],
+            [AS_HELP_STRING([--with-dbussystemconfigdir],
+                            [Path to the system dbus-1/system.d directory])],
+            [dbussystemconfigdir="$withval"], [dbussystemconfigdir="/usr/share/dbus-1/system.d"])
+AC_SUBST(dbussystemconfigdir)
+AC_ARG_WITH([dbussystemservicesdir],
+            [AS_HELP_STRING([--with-dbussystemservicesdir],
+                            [Path to the system dbus-1/system-services directory])],
+            [dbussystemservicesdir="$withval"], [dbussystemservicesdir="/usr/share/dbus-1/system-services"])
+AC_SUBST(dbussystemservicesdir)
+AC_ARG_WITH([xdgautostartdir],
+            [AS_HELP_STRING([--with-xdgautostartdir],
+                            [Path to the system dbus-1 directory])],
+            [xdgautostartdir="$withval"], [xdgautostartdir="/etc/xdg/autostart"])
```

Comment:
Do you need to parametrize this? Could you just use `$sysconfdir/xdg/autostart` in the relevant place?

Reply:
I don't really need to parametrize any of these, I was just following what was being done for the other variables there.